### ISSUE TITLE
Missing function/property to get the ARN of the resource

### ISSUE THREAD

**Issue by Flauschbaellchen** [mooyoul/aws-cdk-ses-domain-identity#8]
Hi,

I think the `DnsValidatedDomainIdentity` class is missing a property for getting the ARN of the created resource, e.g. something like the following:

```
const sesDomain = new DnsValidatedDomainIdentity(this, `example-domain`, {...});
sesDomain.dnsValidatedDomainIdentityArn #=> arn:aws:ses:eu-central-1:0000000000000:identity/example.com
```

I like to use this ARN within an `iam.PolicyStatement` as a resource to limit the action `ses:SendRawEmail` to only the domain I just created.

It can be created manually as only the domain needs to be added, however it would be easier to have this automatism.

**Comment by mooyoul** [comment]
Ah yes. As you've pointed out, ARN is not available now. Will be fixed in next version

**Comment by michaelgmcd** [comment]
Hey @mooyoul, I'd love to see this added. I can help out here if needed. What would it take to add this?

**Comment by michaelgmcd** [comment]
In the meantime, this works:
`arn:aws:ses:${region}:${this.account}:identity/${domainName}`

**Comment by mooyoul** [comment]
Hello all. Sorry for late reply. I just released a new version (v1.1.0) and now you can access ARN via `identityArn` property. Thank you for reporting!
